README.md

```diff
@@ -138,7 +140,9 @@ Not supported but most objects for v2.3 should work for this one too.
 		}
 	])
 	.build()
-	.then(function(bidResponse){
+	.then(function(result){
+		var bidResponse = result.bidResponse //the bid response object
+		var meta = result.meta 	   	     //object that contains an array with possible validation errors
```

Comment:
Meta is a generic object. It will also contain other stuff. So in the comment say that meta is an object holding info about the bid response build process including validation errors